pkg/accounting/tally/tally_test.go

```diff
@@ -55,7 +54,8 @@ func TestOnlineNodes(t *testing.T) {
 	limit := 0
 	interval := time.Second
 
-	tally := newTally(pointerdb, overlayServer, kad, limit, logger, interval)
+	tally, err := newTally(testsql.Driver, testsql.Path(), pointerdb, overlayServer, kad, limit, logger, interval)
```

Comment:
This should suffice `sqlite://file::memory:`, no need for the extra package.

Reply:
If we ever change our SQL setup, it will be convenient to change it in one spot.  Using a specific package conveys meaning to the otherwise unexplained in memory sqlite database.